### DISCUSSION TITLE
一分钟读论文：《关于（未）采用 JavaScript 前端框架的研究》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#17] [Announcements]
# 一分钟读论文：《关于（未）采用 JavaScript 前端框架的研究》

如果你的团队正在考虑迁移新的前端框架，建议你先读巴西南马托格罗索联邦大学计算机系的这篇论文《On the (Un-)Adoption of JavaScript Front-end Frameworks》。作者通过研究 Github 的 TOP-15000 项目，覆盖 Angular、React、VUE、Backbone, Ember 主流框架，对采用或不采用框架有了建设性的发现： 受欢迎程度 (39%) 和易学性 (35%) 是激励采用前端框架的主要因素。 20%的人计划在未来的项目中采用新的的框架。 迁移到新的框架很费时间，41%项目在执行迁移上花费的时间多于使用新旧框架的时间。 在迁移到新框架期间需要更多的开发人员数量。 大多数迁移直接发生在 Master 或 develop 分支中，意味着重写。

https://unbug.github.io/On-the-(Un-)Adoption-of-JavaScript-Front-end-Frameworks/

**Comment by unbug** [comment]
迁移到新框架前先盘点好团队的人力，看看是否有老鸟带，毕竟成本是会增加很多的
